src/draw/text.rs

```diff
@@ -129,6 +129,9 @@ impl Draw for TextState {
                 }
             }
         }
+        if let Some(color) = self.background_color {
```

Comment:
I think it's better not to use an option here and just have:
`if self.background_color != TRANSPARENT {`
You will usually be specifying `Some` anyway since TRANSPARENT is the default. There should be no difference between None and TRANSPARENT, and making it possible to have a value of `Some(TRANSPARENT)` is bad.